### PR TITLE
S3 RETL source IAM role authentication

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added the AWS FTR flow for S3 source doc

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by ruchiramoitra — APPROVED** [review]

File: docs/sources/reverse-etl/amazon-s3.mdx [hunk 1]
```diff
@@ -92,47 +75,122 @@ Your S3 bucket (with the prefix, if specified above) should only consist of Apac
...
+  "s3:GetObject",
+  "s3:ListBucket"
+],
+```
+
+## FAQ
 
 ### Where can I obtain the AWS Access Key ID and the AWS Secret Access Key?
 
 To get the **AWS Access Key ID** and the **Secret Access Key**, follow these steps:
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/amazon-s3.mdx:183]: can remove

File: docs/sources/reverse-etl/amazon-s3.mdx [hunk 2]
```diff
@@ -92,47 +75,122 @@ Your S3 bucket (with the prefix, if specified above) should only consist of Apac
...
+```json
+"Action": [
+  "s3:GetObject",
+  "s3:ListBucket"
+],
+```
+
+## FAQ
 
 ### Where can I obtain the AWS Access Key ID and the AWS Secret Access Key?
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/amazon-s3.mdx:181]: Should we just add the reference link as mentioned in the info block without any steps?

File: docs/sources/reverse-etl/amazon-s3.mdx [hunk 3]
```diff
@@ -92,47 +75,122 @@ Your S3 bucket (with the prefix, if specified above) should only consist of Apac
...
+
+3. Select **Another AWS account** and under **Account ID**, enter `422074288268`, the account ID associated with RudderStack.
+4. Under **Options** check **Require external ID** and enter your <Link to="/resources/glossary/#workspace-id">workspace ID</Link> as the **External ID**.
+
+<img src="../../assets/destinations/aws-role-2.png" alt="Setting up AWS IAM Role for RudderStack" />
+
+5. Review all the settings carefully and click **Next** to proceed.
+6. In the **Permissions** window, select the check box next to the policy you created in the <Link to="#creating-the-policy">Creating the policy</Link> section above.
+7. Review all the settings carefully and click **Next** to proceed.
+8. Enter a unique name for your role. Note that this name **cannot** be distinguished by case. For example, you cannot create a role named `RUDDERSTACK` if `rudderstack` already exists.
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/amazon-s3.mdx:154]:
```suggestion
8. Enter a unique name for your role. Note that this name is **case-sensitive**. For example, you cannot create a role named `RUDDERSTACK` if `rudderstack` already exists.
```

File: docs/sources/reverse-etl/amazon-s3.mdx [hunk 4]
```diff
@@ -3,86 +3,69 @@ title: "Amazon S3"
...
 
-For more information on getting these AWS credentials, refer to the <a href="https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html">AWS documentation</a>.
-</div>
-
-<div class="successBlock">
-
-  If you've already configured S3 as a source before, your existing credentials will automatically appear under <strong>Use Existing Credentials</strong>.
+<div class="warningBlock">
+It is highly recommended to use the <strong>Cross-Account Role</strong> method for connecting to S3 as the <strong>Access Key</strong> method will be deprecated soon.
 </div>
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/amazon-s3.mdx:29]: Should we add S3 permissions link here? As its relevant here but at a lower position in the doc, can be missed?

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**